models/stg_reddit_ads.yml

```diff
@@ -33,7 +33,7 @@ models:
     description: Each record represents a summary of events for each given date and account.
     columns:
       - name: account_id
-        description: '{{ doc("account_id") }}'
+        description: '{{ doc("account_id") }}' 
```

Comment:
Should we also provide a not_null test for this field?

Reply:
updated